types/trace/trace.go

```diff
@@ -37,7 +37,7 @@ type Event struct {
 	PodSandbox          bool         `json:"podSandbox"`
 	EventID             int          `json:"eventId,string"`
 	EventName           string       `json:"eventName"`
-	MatchedScopes       uint64       `json:"matchedScopes"`
+	MatchedPolicies     uint64       `json:"matchedPolicies"`
```

Comment:
@yanivagman a question for a next PR:

Does it make sense to expose in `Event` type a field like `MatchedPoliciesNames []string`?

Reply:
Right. After this merge, I'm going to tackle the ID<=>Name on top of https://github.com/aquasecurity/tracee/pull/2873.